src/Stripe.net/Entities/InvoiceItems/InvoiceItem.cs

```diff
@@ -122,6 +122,13 @@ public Subscription Subscription
         [JsonProperty("tax_rates")]
         public List<TaxRate> TaxRates { get; set; }
 
+        /// <summary>
+        /// Set to <c>true</c> if we grouped proration items into one,
+        /// <c>false</c> if not.
+        /// </summary>
+        [JsonProperty("unified_proration")]
+        public bool UnifiedProration { get; set; }
```

Comment:
@ob-stripe should this be `bool?`? I think it's still partly gated and might not be returned at all.

Reply:
yeah it didn't return for me when I was testing, should probably be `bool?` will wait for ob

Reply:
Making it nullable based on the docs. Happy to switch back if @ob-stripe thinks it should be `bool` instead.

Reply:
According to the spec, it's not nullable. It's fine to declare a field as non-nullable as long as the API does not return a `null` value for it (i.e. either the API returns a `true` or `false` value for the field, or the field is absent from the JSON).

Reply:
ok fixing. thanks!